src/custom/pages/App/index.tsx

```diff
@@ -13,6 +13,7 @@ import RedirectAnySwapAffectedUsers from 'pages/error/AnySwapAffectedUsers/Redir
 import { SENTRY_IGNORED_GP_QUOTE_ERRORS } from 'api/gnosisProtocol/errors/QuoteError'
 
 import SideBanner from 'components/SideBanner'
+const SIDE_BANNER_VISIBLE_KEY = 'isSideBannerVisible'
```

Comment:
nit: i would write storage keys in upper snake case like: COWSWAP_ANNIVERSARY_BANNER or sth

Reply:
Got it. I also stored it in `constants/misc` given it's also used in the  child component.